### ISSUE TITLE
Update 'The Evolution Of Finger' to use endpoints.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8588]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)| @Lukasa reported|
|-|-|
|Trac ID|trac#8588|
|Type|enhancement|
|Created|2016-07-12 08:49:41Z|
|Branch|https://github.com/twisted/twisted/tree/8588-lukasa-finger|

The Twisted Tutorial, 'The Evolution of Finger', uses `internet.TCPServer` and friends. Having a tutorial that relies on such outdated APIs is pretty bad. This issue will fix that up to remove the use of the outdated APIs, and encourage the use of endpoints instead.

<details><summary>Searchable metadata</summary>

```
trac-id__8588 8588
type__enhancement enhancement
reporter__Lukasa Lukasa
priority__normal normal
milestone__None None
branch__8588_lukasa_finger 8588-lukasa-finger
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468313381775071 1468313381775071
changetime__1468779856109744 1468779856109744
version__None None
owner__Lukasa Lukasa

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_1"></a>|@Lukasa commented|
|-|-|

Ok, a patch is available at https://github.com/twisted/twisted/pull/318. What a marathon!

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa commented|
|-|-|

In changeset a261a7f231084a01adb86075f2ac025525896cd1
```
#!CommitTicketReference repository="" revision="a261a7f231084a01adb86075f2ac025525896cd1"
Add a topfile for #8588
```

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_3"></a>|@Lukasa commented|
|-|-|

Please note as well that this is related to [#6923](https://github.com/twisted/twisted/issues/6923).

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set owner to @Lukasa|
|-|-|

Good!  Comments here: https://github.com/twisted/twisted/pull/318#issuecomment-233173484

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 7cc8ebc0becc6abd81a14bb9ac082b559c8062c4
```
#!CommitTicketReference repository="" revision="7cc8ebc0becc6abd81a14bb9ac082b559c8062c4"
Merge 8588-lukasa-finger: Update the Twisted tutorial to use endpoints.

Author: lukasa
Reviewer: rodrigc, hawkowl
Fixes: #8588
```
